airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresDebeziumStateUtil.java

```diff
@@ -214,16 +214,22 @@ private OptionalLong extractLsn(final Set<Partition> partitions,
 
   }
 
+  private static ThreadLocal<JsonNode> initialState = new ThreadLocal<>();
```

Comment:
Note: This is thread local in case some tests are running concurrently.
Otherwise multiple tests might incorrectly share this single instance of initial state .
I don't believe postgres has any, but left it for extra safety.